### ISSUE TITLE
ALE plugin is crashy

### ISSUE THREAD

**Issue by strager** [quick-lint/quick-lint-js#89]
@terite says that the Vim+ALE plugin is crashy, causing exceptions to fly all over the place. Likely this line: https://github.com/quick-lint/quick-lint-js/blob/48a8dd71645de22f9d08039b440af4e15832fb12/plugin/vim/quick-lint-js.vim/autoload/quick_lint_js_ale.vim#L21

ALE version: unknown
Vim version: unknown

**Comment by terite** [comment]
Vim version: NVIM v0.4.4
`quick-lint-js version 0.1.0` from commit `48a8dd7`
ALE version: 64caa507

Error is in the form

```
Error detected while processing function <SNR>86_NeoVimCallback[29]..<lambda>7[1]..<SNR>77_ExitCallback[28]..<SNR>76_HandleExit[24]..quick_lint_js_ale#parse_command_output:
line    6:
E474: Unexpected end of input: {"qflist": [{"col": 1, "lnum": 1, "end_col": 3, "end_lnum": 1, "vcol": 0, "text": "let with no bindings", "bufnr": 1, "filename": "/tmp/nvimpMqa35/4/get.js"},^@{"col": 4, "lnum": 1, "end_col": 4, "end_lnum": 1, "vcol": 0, "text": "missing semicolon after expression", "bufnr": 1, "filename": "/tmp/nvimpMqa35/4/get.js"}
```

When parsing the following file

```js
let get;
```

**Comment by strager** [comment]
This should be fixed in commit 1aba02b3504e896dc14fad35f605d950947d0335 (which I landed in master). I tested with Neovim version v0.4.4 and ALE version 03b6978a270107b670b0363d50f3eed4b365ba26.

Feel free to reopen if the problem still persists after quick-lint-js commit 1aba02b3504e896dc14fad35f605d950947d0335.
